dolphin/alert_manager/trap_receiver.py

```diff
@@ -43,6 +45,106 @@ def __init__(self, trap_receiver_address, trap_receiver_port,
         self.trap_receiver_port = trap_receiver_port
         self.snmp_mib_path = snmp_mib_path
 
+    def sync_snmp_config(self, ctxt, snmp_config_to_del=None,
+                         snmp_config_to_add=None):
+        if snmp_config_to_del is not None:
+            self._delete_snmp_config(ctxt, snmp_config_to_del)
```

Comment:
can return from here

Reply:
No, think about the scenario that update alert_source, we need to remove exist config first, and then add the new config.